src/App.js

```diff
@@ -1,23 +1,43 @@
-import React from "react";
-import { useDispatch, useSelector } from "react-redux";
+import React, { useEffect, useState } from "react";
+import { useSelector } from "react-redux";
 import { CSSTransition } from "react-transition-group";
 import "./App.css";
 import DataBox from "./components/DataBox";
 import InputBox from "./components/InputBox";
 import MyErrorBox from "./components/UI/AlertBox/MyErrorBox";
 import MyDefaultBack from "./components/UI/DefaultBackground/MyDefaultBack";
-import MyVideoBack from "./components/UI/VideoBackground/MyVideoBack";
+import LondonBox from "./components/LondonBox";
+import NewYorkBox from "./components/NewYorkBox";
+import KyivBox from "./components/KyivBox";
+import IpCityBox from "./components/IpCityBox";
+import { useActions } from "./hooks/useActions";
+import TorontoBox from "./components/TorontoBox";
+import MyButton from "./components/UI/Button/MyButton";
 
 function App() {
-  const dispatch = useDispatch();
-  const weatherData = useSelector((state) => state.weatherData);
-  const isError = useSelector((state) => state.isError);
+  const { weatherData, isError } = useSelector((state) => state.weather);
+  const { getMainCitiesWeather } = useActions();
+  const [theme, setTheme] = useState(localStorage.getItem("Theme") || "dark");
 
+  useEffect(() => {
+    getMainCitiesWeather();
+  }, []);
+
+  const handleThemeChange = () => {
+    theme === "light" ? setTheme("dark") : setTheme("light");
+    console.log(theme);
+    localStorage.setItem("Theme", theme === "dark" ? 'light' : 'dark');
```

Comment:
Looks strange for me
It would be clear to me:
    setTheme(theme);
    console.log(theme);
    localStorage.setItem("Theme", theme);